.github/workflows/unit_tests_with_featuretools_main_branch.yml

```diff
@@ -18,22 +18,26 @@ jobs:
           python-version: ${{ matrix.python_version }}
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: Build featuretools package
-        run: make package_evalml
+      - name: Update apt and install Graphviz
+        run: sudo apt update && sudo apt install -y graphviz
       - name: Set up pip
         run: |
           pip config --site set global.progress_bar off
           python -m pip install -U pip
+      - name: Build EvalMl package
+        run: make package_evalml
       - name: Install EvalML & Featuretools - test requirements
         run: |
-          python -m pip install -e unpacked_sdist/[test]
+          cd unpacked_sdist
+          make -f ../Makefile installdeps-prophet
```

Comment:
There is no point in running `make package_evalml` of you are just going to run `installdeps` which just installs from the requirement text files.